mautic/mautic
# Personal and transactional emails do not contain unsubscribe header

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | https://github.com/mautic/user-documentation/pull/136
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR removes an unsubscribe header from an individual and transactional emails. 
[topic in the forum](https://forum.mautic.org/t/personal-email-should-not-contain-an-unsubscribe-header-custom-email/18525/11#post_12)

https://forum.mautic.org/t/personal-email-should-not-contain-an-unsubscribe-header-custom-email/18525/11#post_12


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Send an individual email, an email via API `/emails/ID/contact/CONTACT_ID/send`, transactional email by form and by campaign.
3. Check the headers of the emails. There is no 'List-Unsubscribe'. 
4. Send a marketing email by form and by campaign.
5. Check the headers of the emails. There is 'ListUnsubscribe'.
